litmus-portal/backend/graphql-server/graph/model/models_gen.go

```diff
@@ -3,17 +3,17 @@
 package model
 
 type Cluster struct {
-	ClusterID    string  `json:"cluster_id" bson:"cluster_id"`
-	ProjectID    string  `json:"project_id" bson:"project_id"`
-	ClusterName  string  `json:"cluster_name" bson:"cluster_name"`
-	Description  *string `json:"description" bson:"description"`
-	PlatformName string  `json:"platform_name" bson:"platform_name"`
-	AccessKey    string  `json:"access_key" bson:"access_key"`
-	IsRegistered bool    `json:"is_registered" bson:"is_registered"`
-	IsActive     bool    `json:"is_active" bson:"is_active"`
-	UpdatedAt    string  `json:"updated_at" bson:"updated_at"`
-	CreatedAt    string  `json:"created_at" bson:"created_at"`
-	ClusterType  string  `json:"cluster_type" bson:"cluster_type"`
+	ClusterID    string  `bson:"cluster_id" bson:"cluster_id"`
+	ProjectID    string  `bson:"project_id" bson:"project_id"`
+	ClusterName  string  `bson:"cluster_name" bson:"cluster_name"`
+	Description  *string `bson:"description" bson:"description"`
+	PlatformName string  `bson:"platform_name" bson:"platform_name"`
+	AccessKey    string  `bson:"access_key" bson:"access_key"`
+	IsRegistered bool    `bson:"is_registered" bson:"is_registered"`
+	IsActive     bool    `bson:"is_active" bson:"is_active"`
+	UpdatedAt    string  `bson:"updated_at" bson:"updated_at"`
+	CreatedAt    string  `bson:"created_at" bson:"created_at"`
+	ClusterType  string  `bson:"cluster_type" bson:"cluster_type"`
```

Comment:
Need to change json to bson for mongodb

Reply:
It is already there, the json is for gql and bson is for mongodb. There are 2 tags for the Cluster type.

Reply:
If you don't convert it to BSON, it will remove underscore while inserting data into the database
ex- updated_at will `updatedat` in db.

Check the mongodb after inserting the data

Reply:
this structure is also used by mongodb while inserting and retrieving the data

Reply:
We need to separate MongoDB struct and gql struct because if someones generate the code again bson will be removed automatically.

Reply:
You have changed the tag to just bson, in the previous version I kept both json and bson tags for Cluster type because it is required by both GQL and MongoDB. I believe GQL requires json tags, that's why I had kept both the tags. You can take a look at the older version of this file.

Reply:
I was thinking of separating the two models but the structures are same and as the collections increase a lot of code will be repeated,

Reply:
I am separating mongodb struct from gql struct in this PR